packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -927,7 +927,6 @@ class _DatePickerHeader extends StatelessWidget {
     switch (orientation) {
       case Orientation.portrait:
         return Semantics(
-          container: true,
           child: SizedBox(
```

Comment:
if `container: true,` is removed,  `Semantics` wrapper here can also be removed since it added nothing

Reply:
removed Semantics wrapper